openshift/templates/maintenance/caddy.bc.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
Please convert to yaml to keep it the same as our other templates and configs.

Reply:
uhhh, great. These force pushes were supposed to be on a separate branch. let me clean this up.

Reply:
moved all of this out, will add in it's own PR later.